i18n/management/commands/publish_i18n.py

```diff
@@ -1,16 +1,45 @@
-# pylint: disable=missing-docstring
+# pylint: disable=missing-docstring, broad-except
 import datetime
 import time
+import multiprocessing
 
 from django.conf import settings
 from django.core.management.base import BaseCommand
 from django.utils import translation
+from django import db
 
 from i18n.management.utils import log, get_models_to_sync
 
+
 def can_publish_model(model):
     return hasattr(model, 'publish') or hasattr(model, 'publish_pdfs')
 
+
+def publish(obj):
+    if not obj.should_be_translated:
+        return
+
+    pdf_generation_time = 0
+
+    for language_code in Command.language_codes:
+        translation.activate(language_code)
+        if hasattr(obj, 'publish'):
+            list(obj.publish(silent=True))
+        # Temporary hack to turn off PDF generation while still directing users to the
```

Comment:
nit: I think the comment is out of date and I missed removing it when I removed the "hack"

Reply:
ooooh, that makes sense. I was thinking that this code doesn't look like a hack